datanode/gateway/graphql/resolvers.go

```diff
@@ -1654,6 +1650,26 @@ func (r *myQueryResolver) ReferralSetReferees(ctx context.Context, id, referrer,
 	return resp.ReferralSetReferees, nil
 }
 
+func (r *myQueryResolver) ReferralSetStats(ctx context.Context, setID string, epoch *int, partyID *string, pagination *v2.Pagination) (*v2.ReferralSetStatsConnection, error) {
+	var epochU64Ptr *uint64
+	if epoch != nil {
+		epochU64 := uint64(*epoch)
+		epochU64Ptr = &epochU64
+	}
+
+	resp, err := r.tradingDataClientV2.GetReferralSetStats(ctx, &v2.GetReferralSetStatsRequest{
+		ReferralSetId: setID,
+		AtEpoch:       epochU64Ptr,
+		Referee:       partyID,
+		Pagination:    pagination,
+	})
+	if err != nil {
+		return &v2.ReferralSetStatsConnection{}, err
```

Comment:
Why return a pointer to a valid, but uninitialised type instead of `return nil, err`?

Reply:
A mistake.